src/plugins/Dashboard/FileItem.js

```diff
@@ -99,7 +99,7 @@ module.exports = function fileItem (props) {
         }
       </h4>
       <div class="UppyDashboardItem-status">
-        ${file.data.size && html`<div class="UppyDashboardItem-statusSize">${prettyBytes(file.data.size)}</div>`}
+        ${isNaN(file.data.size) ? '' : html`<div class="UppyDashboardItem-statusSize">${prettyBytes(file.data.size)}</div>`}
```

Comment:
We are trying to support IE 10-11, so we’ll need a polyfill for this one, I think.

Reply:
We could use lodash.isFinite for this, but again I'm not sure if they support IE10. I could just hardcore something like [this](https://gist.github.com/dfkaye/98e21e95c8fe528667cf) but it looks messy.